CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.13.2
+
+* `FIX`: remove unnecessary scroll padding ([#145](https://github.com/bpmn-io/properties-panel/pull/145))
+* `FIX`: keep existing configuration after template apply ([#661](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/661))
+* `FIX`: always override `hidden` configuration on template apply ([#661](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/661))
+* `FIX`: do not render non-existing values in element template ([#676](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/676))
+* `FIX`: pick-up correct element template icon ([#670](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/670))
+* `CHORE`: update to `bpmn-js-properties-panel@1.1.1`
+* `CHORE`: update to `@bpmn-io/properties-panel@0.13.2`
```

Comment:
Let's scope the changes to differentiate what we did in camunda-bpmn-js and what changes come from the dependency updates (cf. `0.13.0-alpha.8` changes):

```suggestion
* `CHORE`: update to `bpmn-js-properties-panel@1.1.1`
* `CHORE`: update to `@bpmn-io/properties-panel@0.13.2`

### Key changes in Properties Panel

* `FIX`: remove unnecessary scroll padding ([#145](https://github.com/bpmn-io/properties-panel/pull/145))
* `FIX`: keep existing configuration after template apply ([#661](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/661))
* `FIX`: always override `hidden` configuration on template apply ([#661](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/661))
* `FIX`: do not render non-existing values in element template ([#676](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/676))
* `FIX`: pick-up correct element template icon ([#670](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/670))
```

Reply:
Makes a lot of sense, thanks for suggesting.